.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - "8.9.4"
+  - node
```

Comment:
Do you need quotes here? I mean `"node"`?

Reply:
Quotes are not required. [Check the travis ci build here](https://travis-ci.org/mrdoob/three.js/builds/429154259#L436)